.changeset/sharp-ducks-whisper.md

```diff
@@ -0,0 +1,6 @@
+---
+"@livekit/components-core": minor
+"@livekit/components-react": minor
+---
+
+Update more hooks and components to use `trackRef` and participant/source property pairs
```

Comment:
Did you mean "...instead of participant/source property pairs"?

Reply:
"deprecate"